angular/projects/ng-kinintel/src/lib/components/dataset/dataset-editor/dataset-filters/dataset-filter/dataset-filter.component.ts

```diff
@@ -99,6 +100,12 @@ export class DatasetFilterComponent implements OnInit {
                 filter.rhsExpression[1] = 1;
             }
         }
+        if (filter.filterType === 'like' || filter.filterType === 'notlike') {
+            if (!filter.rhsExpression[1]) {
+                filter.rhsExpression[1] = 'likewildcard';
+            }
+            console.log(filter);
```

Comment:
Console log in prod??

Reply:
Also testing with expressions like:
RLIKE 'like'
RLIKE 1
RLIKE 'w{3}'
RLIKE 'google\.com'